packages/app/src/runner/selector-playground/SelectorPlayground.cy.tsx

```diff
@@ -4,6 +4,14 @@ import { Clipboard_CopyToClipboardDocument } from '../../generated/graphql-test'
 import SelectorPlayground from './SelectorPlayground.vue'
 import { logger } from '../logger'
 
+/**
+ * Helper to reset focus for tooltips
+ * jQuery .blur() seemed unreliable, leading to flake
+ */
+function clickAway () {
```

Comment:
It would be good to understand more how this fixes the problem better than just the `blur` does.  If this is the best way to handle tooltip dismissals, then perhaps the concept could be encapsulated in a custom `cy` command.

Reply:
If the intent is to ensure all tooltips are closed and lose focus there is a special function provided by our chosen tooltip library that allows you to close all open instances - would introduce tighter coupling to the impl but we're already using it in our `StandardModal.vue` 🤷‍♂️ 

```
import { hideAllPoppers } from 'floating-vue'
...
hideAllPoppers()
```

Just floating as a possibility, probably overkill for this scenario

Reply:
A custom command would make sense in tackling this more broadly.

I can think of a few ways to tease out what is going on, maybe by displaying the mouse position or something, but that's probably a separate task.